Kernel/Process.cpp

```diff
@@ -2726,6 +2727,28 @@ int Process::sys$reboot()
     return ESUCCESS;
 }
 
+int Process::sys$mount(int devmajor, int devminor, const char* mountpoint)
```

Comment:
A better prototype would be `sys$mount(const char* device, const char* mountpoint)`
We can iterate on this though.

Reply:
This is because I was doing `stat` in the program. Whoops!